kolibri/plugins/device/assets/src/views/ManageContentPage/channelUpdateStrings.js

```diff
@@ -0,0 +1,45 @@
+import { createTranslator } from 'kolibri.utils.i18n';
+
+export default createTranslator('ChannelUpdateStrings', {
+  notAvailableFromDrives: 'This channel was not found on any attached drives',
+  notAvailableFromNetwork: 'This channel was not found on other instances of Kolibri',
+  notAvailableFromStudio: 'This channel was not found on Kolibri Studio',
+  newVersionAvailableHeader: "Version {version} of '{channel}' is available",
+  currentVersionLabel: 'You are currently on version {version}',
```

Comment:
These messages are in NewChannelVersionPage, but have different keys and slightly different punctuation. e.g. `New resources available:` (has a colon at the end).